### PR TITLE
Feature/trip files

### PR DESCRIPTION
**Pull request by emilyjmiles**
## Type of change

- [ ] Bug fix (fixes an issue)
- [x] New feature (adds functionality)
- [ ] Refactor

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

- [ ] I tested my changes in the browser
- [x] I tested with Mocha/Chai

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new bugs
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Is your code D.R.Y?
- [x] Does it follow SRP?

### CODE REVIEW

File: src/classes/Trip.js [hunk 1]
```diff
@@ -4,7 +4,36 @@ class Trip {
     this.userTrips = tripData.filter(trip => trip.userID === userID);
     this.trips = tripData;
     this.destinations = destinationData;
-  }
-}
+  };
+
+  getTripStatus(tripStatus) {
+    const filteredTrips = this.userTrips
+      .filter(trip => trip.status === tripStatus);
```
Comment (willhobson85) [inline src/classes/Trip.js:13]: Empty line before return

**Review by willhobson85 — CHANGES_REQUESTED** [review]
Hi Emily. Everything looks fantastic. I only saw one small edit, a missing blank line. Your testing is solid.

File: src/classes/Trip.js (hunk 1 above)
Comment (emilyjmiles) [reply src/classes/Trip.js:13]: I have added the white space in the getTripStatus method and also added an additional method to get the total cost for a given year

**Review by willhobson85 — APPROVED** [review]
This looks fantastic! Good work with dayJS!

**Pull request merged**